cmd/cloud/workspace.go

```diff
@@ -52,6 +54,80 @@ func newWorkspaceSwitchCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newWorkspaceUserRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "user",
+		Aliases: []string{"us"},
+		Short:   "Manage users in your Astro Workspace",
+		Long:    "Manage users in your Astro Workspace.",
+	}
+	cmd.SetOut(out)
+	cmd.AddCommand(
+		newWorkspaceUserListCmd(out),
+		newWorkspaceUserUpdateCmd(out),
+		newWorkspaceUserRemoveCmd(out),
+		newWorkspaceUserAddCmd(out),
+	)
+	return cmd
+}
+
+func newWorkspaceUserAddCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add [email]",
+		Short: "Add a user to an Astro Workspace with a specific role",
+		Long: "Add a user to an Astro Workspace with a specific role\n$astro workspace user add [email] --role [WORKSPACE_MEMBER, " +
+			"WORKSPACE_BILLING_ADMIN, WORKSPACE_OWNER].",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return addWorkspaceUsers(cmd, args, out)
```

Comment:
Should these function name be singular..`addWorkspaceUser`, `updateWorkspaceUser`, `removeWorkspaceUser`

Reply:
works for me